specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-11-01/virtualMachine.json

```diff
@@ -785,7 +785,7 @@
           "VirtualMachines"
         ],
         "operationId": "VirtualMachines_Delete",
-        "description": "The operation to delete a virtual machine.",
+        "description": "The operation to delete a virtual machine. NOTE: As of version 2024-11-01, we are rolling out a feature where if the forceDeletion parameter is unspecified OR not explicitly set to false, AND all of the VM's attached disks including the OS disk are marked with the delete option, then the VM will be force deleted. For more details on how to configure delete options for a VM's resources, see [Delete a VM and attached resources](https://learn.microsoft.com/en-us/azure/virtual-machines/delete). To avoid defaulting to force delete, ensure that the forceDeletion parameter is explicitly set to false. This feature is expected to rollout by end of March 2025.",
```

Comment:
Add <br><br> before "NOTE:" and also change version to "api-version:" and the comment should be added to description of the forceDeletion parameter.